draft-barnes-hpke-hpke.md

```diff
@@ -1122,16 +1206,18 @@ algorithm whose output length is `Npk`.
 
 ## Key Derivation Functions (KDFs) {#kdf-ids}
 
-| Value  | KDF         | Nh  | Reference    |
-|:-------|:------------|-----|:-------------|
-| 0x0000 | Reserved    | N/A | RFC 9180     |
-| 0x0001 | HKDF-SHA256 | 32  | {{?RFC5869}} |
-| 0x0002 | HKDF-SHA384 | 48  | {{?RFC5869}} |
-| 0x0003 | HKDF-SHA512 | 64  | {{?RFC5869}} |
+| Value  | KDF         | Nh  | Two-Stage | Reference    |
+|:-------|:------------|-----|-----------|:-------------|
+| 0x0000 | Reserved    | N/A | N/A       | RFC 9180     |
+| 0x0001 | HKDF-SHA256 | 32  | Y         | {{?RFC5869}} |
+| 0x0002 | HKDF-SHA384 | 48  | Y         | {{?RFC5869}} |
+| 0x0003 | HKDF-SHA512 | 64  | Y         | {{?RFC5869}} |
 {: #kdfid-values title="KDF IDs"}
 
 ### Input Length Restrictions {#kdf-input-length}
 
+[[ TODO(RLB): Determine what this section should say about single-stage KDFs. ]]
```

Comment:
Seems like psk, psk_id, info and ikm are limited to 65535 bytes for single-stage KDFs. Exporter_context has no explicit limit, but may be limited by total Derive() input length. For underlying constructs related to SHA-3, there are no limits on exporter_context.